src/main/java/org/jabref/gui/entryeditor/EntryEditor.css

```diff
@@ -31,6 +31,12 @@
     -fx-background-color: -jr-icon-background-active;
 }
 
+#typeLabel .context-menu,
+#typeLabel .tooltip {
+    -fx-font-size: 13px;
```

Comment:
Does `-fx-font-size: 1em` also work? Generally, it is best to avoid font sizes in `px` because then a) the actual size depends on the font size and b) they don't scale if the global font size is changed.

Reply:
I checked with `1em` at the beginning, but it did not work. However, I agree with you: it is so better to use relative sizes. Finally, I have been trying many values, and now it is done.

Reply:
Perfect, thanks.